app/src/main/java/com/example/sopt30/ProfileFragment.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.sopt30
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.bumptech.glide.Glide
+import com.example.sopt30.databinding.FragmentFollowerListBinding
+import com.example.sopt30.databinding.FragmentProfileBinding
+
+class ProfileFragment : Fragment() {
+    private var _binding : FragmentProfileBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        _binding = FragmentProfileBinding.inflate(layoutInflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initTransactionAction()
+        initImage()
+    }
+
+    private fun initImage() {
+        Glide.with(this)
+            .load("https://s3.orbi.kr/data/file/united/957fd87b5938e6af46779e971012aa6d.jpg")
+            .circleCrop()
+            .into(binding.ivProfile)
+    }
+
+    private fun initTransactionAction() {
+        val followerFragment = FollowerListFragment()
+        val repositoryFragment = RepositoryListFragment()
+
+        childFragmentManager.beginTransaction().add(R.id.fcv_list, followerFragment).commit()
```

Comment:
fragment-ktx 이용해서 아래처럼 프래그먼트 클래스만 참조해서 보다 간결하게 표현이 가능합니다..!
```suggestion
childFragmentManager.commit {
    replace<FollowerFragment>(R.id.fcv_list)
}
```

Reply:
메모리 릭에도 세이프하고 말이죵

Reply:
오왕 fragment-ktx는 처음알았네요!